app/views/layouts/application.html.erb

```diff
@@ -14,16 +14,16 @@
         <li>
           <%= link_to "Logout", destroy_user_session_path, method: :delete%>
```

Comment:
This may be picky, but if we're going to clean up this file, could you add a space on line 15 between `:delete` and  `%>`